WalletWasabi.Gui/Controls/WalletExplorer/CoinViewModel.cs

```diff
@@ -131,7 +135,7 @@ public void UnsubscribeEvents()
 		public string Address => Model.ScriptPubKey.GetDestinationAddress(Global.Network).ToString();
 
 		public int Confirmations => Model.Height.Type == HeightType.Chain
-			? Global.BitcoinStore.HashChain.TipHeight - Model.Height.Value + 1
+			? Global.BitcoinStore.HashChain.TipHeight - Height + 1
```

Comment:
What does this do?

Reply:
Not sure it's wise to call `- Height + 1`. Would prefer `Height.Value` we had issues previously with `+-=`, etc operations on the `Height`. Maybe if you want to write unit tests for that, it'd be ok. Anyway, how does it fix anything?